.circleci/config.yml

```diff
@@ -71,7 +77,9 @@ jobs:
       - checkout
       - run:
           name: Run tests
-          command: python3 setup.py test -P visualization --remote-data=astropy --open-files -a "--mpl --mpl-results-path=$PWD/results"
+          command: |
+            pip install pytest-astropy-header --upgrade --no-deps
```

Comment:
why is this necessary, isn't it added to `pytest-astropy`? Or just a temporary measure for now until there is a new upstream release?

Reply:
It's temporary because pytest-astropy is already installed in the docker images but is an old version..